jsonnet/utils/sanitize-rules.libsonnet

```diff
@@ -326,6 +326,26 @@ local patchedRules = [
   },
 ];
 
+local addedRules = [
```

Comment:
Can this be defined in upstream prometheus mixin?

Reply:
Yes, I will submit a change to upstream prometheus mixins later. Once it is fit into upstream, we can clean it from CMO.

Reply:
PR created in Prometheus to add this alert to mixins: https://github.com/prometheus/prometheus/pull/9873
Hope it can be merged someday :)

Reply:
Can you add a TODO so we don't forget?